packages/wonder-blocks-layout/util/util.test.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import {queryMatchesSize} from "./util.js";
+
+import type {MediaQuery, MediaSize} from "./types.js";
+
+const testQueryMatchesSizes = (
+    query: MediaQuery,
+    matchedSizes: Set<MediaSize>,
+) => {
+    describe(query, () => {
+        for (const size of ["small", "medium", "large"]) {
+            // Arrange
+            const expectedResult = matchedSizes.has(size);
+
+            it(`${query} should ${
+                expectedResult ? "" : "not "
+            }match ${size}`, () => {
+                // Act
+                const actualResult = queryMatchesSize(query, size);
+
+                // Assert
+                expect(actualResult).toEqual(expectedResult);
+            });
+        }
+    });
+};
```

Comment:
I'm really not keen on this kind of test-case generation.

Usually, we don't need to test every single possible combination and I don't know that it really is helpful when tests fail. For example, if only one case fails, it's really hard to investigate that case in isolation.

Instead, could you write each case out itself. It will be clearer what's happening for folks working on this code and though it means a lot of very similar code, it will be easier to maintain.